docs/attributes-registry/system.md

```diff
@@ -0,0 +1,160 @@
+# System
+
+<!-- toc -->
+
+- [CPU attributes](#cpu-attributes)
+- [Memory attributes](#memory-attributes)
+- [Paging attributes](#paging-attributes)
+- [Filesystem attributes](#filesystem-attributes)
+- [Network attributes](#network-attributes)
+- [Process attributes](#process-attributes)
+
```

Comment:
there are also existing deprecated system attributes, available already in registry. Could you add them here as well in the end?

Reply:
Yes, added it.